PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -19,6 +20,12 @@
 protonTable.tagRecoProtonsSingle = cms.InputTag("filteredProtons" if singleRPProtons else "ctppsProtons","singleRP")
 
 
+genProtonTable = _genproton.clone(
+    cut = cms.string('(pdgId == 2212) && (abs(pz) > 5200) && (abs(pz) < 6467.5)'), # xi in [0.015, 0.2]
+    # PU protons source could have an extra 'genPUProtons' collection name for some samples
+    # srcPUProtons = cms.InputTag('genPUProtons', 'genPUProtons'),
```

Comment:
are those comments line meaningful now ?
better to remove this commented code, if not needed

Reply:
For the time being, as several privately-produced samples are relying on this patch we just left it for bookkeeping. If needed we can introduce a customisation command for `cmsDriver.py` instead?